Makefile

```diff
@@ -145,3 +145,8 @@ endif
 .PHONY: local-setup
 local-setup: kustomize kind
 	utils/local-setup.sh
+
+# Rebuild and push the docker image and redeploy authorino to the local k8s cluster
+.PHONY: local-deploy
```

Comment:
Happy to rename it in case you think it's confusing with `local-setup`.

Reply:
Let's review both tasks after some use

Reply:
`make local-image-reload`?

Reply:
`make local-image`?

Reply:
what about `make build-to-kind` ? 

This script will end up, building the images and loading them into the local Kind cluster, not only making images.

Reply:
> what about make build-to-kind ?

I think better keeping it consistent with the other one, `local-setup`. It seems that "local" already means "to kind".